novacustom/v5x0tu/MeteorLakeFspBinPkg/FspPkgPcdShare.dsc

```diff
@@ -0,0 +1,114 @@
+## @file
+#  Platform description for DynamicEx PCDs, defined in FSP Package
+#  and shared with Board Package.
+#
+# @copyright
+#  INTEL CONFIDENTIAL
```

Comment:
This doesn't look good. I can see not all of the files have the confidential mark.

Also let's not add files that are not used anywhere.

Reply:
Maybe bug report to upstream?

Reply:
These files comes from Intel R&DC @pietrushnic

Reply:
All files containing this mark have been removed.